### PR TITLE
DOCS-2430: Merge autogen: robot

### PR DESCRIPTION
**Pull request by andf-viam**
Merge `robot` (robot client) generated content.

Noticed:
- Go > Refresh not found upstream in the [usual place](https://pkg.go.dev/go.viam.com/rdk/robot), but found in docs (and [here](https://pkg.go.dev/go.viam.com/rdk/robot/client#RobotClient.Refresh)). Omitting for now until we can determine what non-interface methods should also be included.

### CODE REVIEW

File: static/include/robot/apis/generated/robot.md [hunk 1]
```diff
@@ -0,0 +1,760 @@
...
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `api_key` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): your API key
+- `api_key_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): your API key ID. Must be a valid UUID
+
+**Returns:**
+
+- ([typing_extensions.Self](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient)): the RobotClient.Options
```
Comment (npentrel) [inline static/include/robot/apis/generated/robot.md:576]: This is a little odd. Wonder if we should replace with `viam.robot.client.RobotClient` or just `RobotClient`

File: static/include/robot/apis/generated/robot.md [hunk 2]
```diff
@@ -0,0 +1,760 @@
...
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.log).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetCloudMetadata
```
Comment (npentrel) [inline static/include/robot/apis/generated/robot.md:518]: missing python code example

**Review by npentrel — APPROVED** [review]
2 small things - up to you if you want to merge & file tickets, or fix and then merge

File: static/include/robot/apis/generated/robot.md (hunk 2 above)
Comment (andf-viam) [reply static/include/robot/apis/generated/robot.md:518]: There [isn't one](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.get_cloud_metadata). I've added manually from what is live in docs currently.

File: static/include/robot/apis/generated/robot.md (hunk 1 above)
Comment (andf-viam) [reply static/include/robot/apis/generated/robot.md]: Guess: `typing_extensions.Self` is relative to calling function, could be more than just `RobotClient` elsewhere. For now, I want to match to only case I've found (for `robot` resource), but expecting to have to walk (some of) this guess back later. For now, this is my best balance: return data type will always be correct (as-scraped), while possibility of incorrect link target exists.

If I complete all autogen merges and this return data type is only used for `RobotClient`, I will indeed replace the data type name everywhere.

**Pull request merged**